ledger/testing/randomAccounts.go

```diff
@@ -100,7 +100,7 @@ func RandomAssetParams() basics.AssetParams {
 		DefaultFrozen: crypto.RandUint64()%2 == 0,
 	}
 	if crypto.RandUint64()%5 != 0 {
-		ap.UnitName = fmt.Sprintf("un%x", uint32(crypto.RandUint64()%0x7fffffff))
+		ap.UnitName = fmt.Sprintf("un%x", uint32(crypto.RandUint64()%0x7fffff))
```

Comment:
This is necessary because otherwise it generates a string that's 10 characters long rather than the 8 allowed by the `MaxAssetUnitNameBytes`

Reply:
I know you did not write the conditional, but do you happen to know why we check if the random int is divisible by 5?

Reply:
It seems like we want only 4/5s of generated AssetParams to have these fields defined? I don't know though